Samples~/SampleGame/Assets/Scripts/Server/GameLiftServer.cs

```diff
@@ -204,7 +229,7 @@ private ProcessParameters CreateProcessParameters()
                 return true;
             },
             _port, // tell the GameLift service which port to connect to this process on.
```

Comment:
Your linter is getting confused by these comments, move them above port so they're at the same indent level